panther_controller/README.md

```diff
@@ -1,122 +1,75 @@
+| `drive_controller`        | Manages mobile robots with a differential drive. It converts velocity commands for the robot body into wheel commands for the base. It also calculates odometry from hardware feedback and shares it. <br/> *[diff_drive_controller/diff_drive_controller](https://github.com/ros-controls/ros2_controllers/tree/master/diff_drive_controller)*                                                                                                                                                                                                                                                                                        |
+| `imu_broadcaster`         | Publishes readings of IMU sensors. <br/> *[imu_sensor_broadcaster/imu_sensor_broadcaster](https://github.com/ros-controls/ros2_controllers/tree/master/imu_sensor_broadcaster)*                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| `joint_state_broadcaster` | Reads all state interfaces and reports them on specific topics. <br/> *[joint_state_broadcaster/joint_state_broadcaster](https://github.com/ros-controls/ros2_controllers/tree/master/joint_state_broadcaster)*                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| `robot_state_publisher`   | Broadcasts a robot's state to tf2 using a provided URDF model and joint states. It updates the model and broadcasts poses for fixed and movable joints to tf2 topics. <br/> *[robot_state_publisher/robot_state_publisher](https://github.com/ros/robot_state_publisher)*                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
I agree, I was thinking about moving it to the main ROS_API.md, but there are very big differences between the description of the nodes on the real robot and in the simulation. The question is, can you see anything better?

Reply:
Right now this info is in panther_bringup, panther_gazebo and in each package. I would remove this table from individual packages.
Can you elaborate on those differences between simulation and hardware and why it can't be moved to ROS_API?

Reply:
It can be moved, I just looking for something better, or aproval moving this to ROS_API.md